test/modules/OS/POSIX/pipe.chpl

```diff
@@ -0,0 +1,27 @@
+use OS.POSIX;
+
+var fildes:2*c_int;
```

Comment:
I don't think we always guarantee that tuple variables are stored contiguously in memory. So it would be better to use c_array here to be more future-proof. I do expect this will work as a global variable like here.

Reply:
Hmm. I'd used a tuple because of the advice in https://chapel-lang.org/docs/language/spec/interoperability.html#referring-to-external-c-types. But it would be easy to change this to use `c_array` instead. What's your preference?

Reply:
My preference would be to use a `c_array` but I wouldn't view it as a show-stopper here by any means.

Reply:
Agreed, module and test changes done in https://github.com/chapel-lang/chapel/pull/19632/commits/8138780da7db75b6e8c0471f8aaa108fdea63a7c.